synapse/tests/test_lib_rstorm.py

```diff
@@ -223,6 +223,102 @@
 Bye!
 '''
 
+shell_input00 = '''
+Shell with environment variable.
+
+.. shell-env:: SYN_LOG_LEVEL=DEBUG SYN_FOO=BAR
+.. shell:: python3 -c "import os; print('LEVEL', os.environ.get('SYN_LOG_LEVEL')); print('FOO', os.environ.get('SYN_FOO'))"
+.. shell-env::
```

Comment:
```suggestion
.. This resets shell-env directives
.. shell-env::
```

Reply:
I added a comment to the shell-env docstring instead